otter/tap/api.py

```diff
@@ -227,6 +228,17 @@ def makeService(config):
         admin_service = service(str(admin_port), admin_site)
         admin_service.setServiceParent(s)
 
+    # setup cloud feed
+    cf_conf = config.get('cloudfeeds', None)
+    if cf_conf is not None:
+        service_configs[ServiceType.CLOUD_FEEDS] = {
```

Comment:
Why are you modifying `service_configs` here instead of just making `get_service_configs` return it?